app/Http/Controllers/UserController.php

```diff
@@ -11,7 +11,7 @@ public function update(User $user, UpdateUserRequest $request)
     {
         // TASK: change this line to not allow is_admin field to be updated
         // Update only the fields that are validated in UpdateUserRequest
-        $user->update($request->all());
+        $user->update($request->except('is_admin'));
```

Comment:
Yes this is correct but there are other alternatives.

For example: `$requtest->validated();`